### DISCUSSION TITLE
Relationship with https://github.com/toland/qlmarkdown

### DISCUSSION THREAD

**Discussion by twwwt** [sbarex/QLMarkdown#90] [Q&A] [answered]
While upgrading Homebrew today, a strange version upgrade of `qlmarkdown 1.3.6 -> 1.0.9` caught my eyes -- an upgrade from a version `1.3.6` to a version `1.0.9` is not what I would consider sane (but that's another story that is more related to Homebrew). After digging a little deeper, I noticed that version `1.3.6` of the Homebrew cask named *qlmarkdown* actually referred to https://github.com/toland/qlmarkdown, while version `1.0.9` refers to this project.

Is there any relationship between https://github.com/toland/qlmarkdown and this project, like for instance that this project has been derived from the former; or are they completely different and the fact that they are named equally is just a coincidence?

**Comment by twwwt** [comment, marked as answer]
Uups, sorry for the noise. I've just noticed https://github.com/toland/qlmarkdown/issues/102

Consider this question answered.

**Comment by sbarex** [comment]
They are two completely different projects. 

https://github.com/toland/qlmarkdown uses the old Quick Look APIs and has not been updated for two years. The markdown engine is Discount.

My app uses the new Quick Look API, and the cmark-gfm engine.

The name is the same for a trivial reason: **QLMarkdown** = **Q**uick **L**ook **Markdown**
